src/HelloTriangle.cpp

```diff
@@ -333,7 +339,10 @@ int helloTriangle() try{
 	std::tie( graphicsQueueFamily, presentQueueFamily ) = getQueueFamilies( physicalDevice, surface );
 
 	const VkPhysicalDeviceFeatures features = {}; // don't need any special feature for this demo
-	const vector<const char*> deviceExtensions = { VK_KHR_SWAPCHAIN_EXTENSION_NAME };
+	vector<const char*> deviceExtensions = { VK_KHR_SWAPCHAIN_EXTENSION_NAME };
+#if __APPLE__
+	deviceExtensions.push_back(VK_KHR_PORTABILITY_SUBSET_EXTENSION_NAME);
+#endif

```

Comment:
As said above, it might not be needed for our purposes? We are not really querying anything using the extension or supplying required features from it to `vkCreateDevice`.

Reply:
Thanks, updated